packages/suite/src/types/suite/guide.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+export type GuideView = 'GUIDE_DEFAULT' | 'GUIDE_CATEGORY' | 'GUIDE_ARTICLE';
+
+export type FeedbackView = 'FEEDBACK_TYPE_SELECTION' | 'FEEDBACK_BUG' | 'FEEDBACK_SUGGESTION';
+
+export type ActiveView = GuideView | FeedbackView;
+
+export type Category =
+    | 'dashboard'
+    | 'account'
+    | 'settings'
+    | 'send'
+    | 'receive'
+    | 'trade'
+    | 'other';
+export type Rating = 1 | 2 | 3 | 4 | 5;
+
+export type FeedbackType = 'BUG' | 'SUGGESTION';
+
+export interface UserDataCommon {
+    platform: 'web' | 'desktop';
+    os: string;
+    user_agent: string;
+    suite_version: string;
+    suite_revision: string;
+    screen_dimensions: string;
```

Comment:
IMHO these screen_dimensions do not tell us anything. 
I think it is better to use window width and height or both.

Reply:
Let's refine these it in the next version